test/suite/test_dump03.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
```

Comment:
Consider merging this new test file into the existing test_dump02.py and covering all the permutations of the 3 new command args "-k", "-n", and "-w".

Reply:
I'm not against it, but time wise not now.